.editorconfig

```diff
@@ -0,0 +1,12 @@
+root=true
+
+[*]
+charset=utf-8
+end_of_line=lf
+insert_final_newline=true
+indent_style=space
+indent_size=4
+
+[*.yml]
```

Comment:
We don't have yaml.

Reply:
Yes, but this project has travis.yml. I hesitated to include or not the settings for it. It may not be worth it, but this file will be reformatted in case auto-formatting is enabled in the IDE. I can exclude this.